lib/account-manager.js

```diff
@@ -135,6 +135,18 @@ module.exports = (challengeManager, config, random, sessionManager, storageServi
 
 
     /**
+     * Destroy the session record.
+     *
+     * @param {string} accountId Account ID
```

Comment:
Do we need a comment saying Account ID when the parameter is `accountId`? I feel this is self-explanatory. Same with Session ID below this.